tests/hub/test_hub_scan_cache.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Alibaba, Inc. and its affiliates.
+import unittest
+
+from modelscope.hub.cache_manager import scan_cache_dir
+from modelscope.hub.errors import CacheNotFound
+from modelscope.utils.file_utils import get_modelscope_cache_dir
+from modelscope.utils.logger import get_logger
+
+logger = get_logger()
+
+
+class HubScanCacheTest(unittest.TestCase):
+
+    def setUp(self):
+        """Set up for tests."""
+        self.fake_cache_dir = '/fake/cache/path'
+
+    def test_scan_default_dir(self):
```

Comment:
we should ensure that at least one model exists, by downloading a known model first (it can be one with only readme file) in the test